ui/src/bin/copy_data.js

```diff
@@ -93,19 +104,63 @@ const run = async () => {
 				subtitle: () => spec.subtitle,
 				title: () => spec.title,
```

Comment:
I imagine many users will be using Excel to look at these data dumps so could be good to move this up to be the second column. It will be more visible when viewed in the UI and gives more info than the `id` column.

Reply:
Agreed for `title` but are we sure we want to move `subtitle` up too? These are usually quite long so that opening the CSV in a spreadsheet you'd have to scroll on the right to see the remaining fields.

Reply:
Oh sorry, I only meant to highlight the line with `title`. I think leave `subtitle` at the end.

Reply:
perfect thanks